1.0/docs/devguide/events.md

```diff
@@ -286,19 +267,19 @@ Example with `listeners`:
                 break;
             }
           }
-
         });
-
       </script>
-
     </dom-module>
 
 
 ## Event retargeting {#retargeting}
 
 Shadow DOM has a feature called "event retargeting" which changes an event's
-target as it bubbles up, such that target is always in the receiving element's
-light DOM. Shady DOM does not do event retargeting, so events may behave differently
+target as it bubbles up, such that target is always in the same scope as the
+receiving element. (For example, for a listener in the main document, the
+target is an element in the main document, not in a shadow tree.)
+
+Shady DOM does not do event retargeting, so events may behave differently
```

Comment:
When I read this I asked myself "why is there a difference between the two?" Could be helpful to add a line addressing that.